RELEASE.md

```diff
@@ -101,6 +101,14 @@ Fix incompatibilities if any.
 * [tools/win32/configure.ps1](tools/win32/configure.ps1)
 * [tools/win32/configure-dev.ps1](tools/win32/configure-dev.ps1)
 
+#### Re-provision Build Server
+
+Even if there aren't any new releases of dependencies with versions
+hardcoded in the repos and files listed above (Boost, OpenSSL).
+
+There may be new build versions of other dependencies (VS, MSVC).
```

Comment:
I don't see why this is split over two paragraphs. Sounds like a single sentence to me.